### ISSUE TITLE
Example unable to be used

### ISSUE THREAD

**Issue by danielhjacobs** [mdn/content#37392]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/ScreenOrientation/lock

### What specific section or headline is this issue about?

Examples

### What information was incorrect, unhelpful, or incomplete?

Clicking fullscreen prints TypeError: Disallowed by permissions policy

Clicking Lock prints SecurityError: Failed to execute 'lock' on 'ScreenOrientation': The window is sandboxed and lacks the 'allow-orientation-lock' flag.

### What did you expect to see?

Usable example

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/screenorientation/lock`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/ScreenOrientation/lock
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/screenorientation/lock/index.md
* Last commit: https://github.com/mdn/content/commit/0e825c9d36e8598691198a6cd3983c4c860ba42a
* Document last modified: 2024-06-30T15:53:38.000Z

</details>

**Comment by skyclouds2001** [comment]
I think we should find some ways to specified sandbox flags and allow attribute and iframe with live examples...

**Comment by wbamberg** [comment]
Does param $6 of the macro: https://github.com/mdn/yari/blob/main/kumascript/macros/EmbedLiveSample.ejs#L13 help here?

**Comment by skyclouds2001** [comment]
> I think we should find some ways to specified sandbox attribute on iframe with live examples...

No, for this case, a `sandbox="allow-orientation-lock"` flag is required for this sample

**Comment by danielhjacobs** [comment]
So would the change be this (in yari)? 
```diff
var allowedFeatures = $6;
+const sandboxAttributes = $7;
```
Can also be `var` just to match the other variables but `const` is modern.
```diff
-%> sandbox="allow-same-origin allow-scripts"></iframe></div><%
+%> sandbox="allow-same-origin allow-scripts<%= $7 ? ' ' + $7 : '' %>"></iframe></div><%
```

**Comment by caugner** [comment]
> So would the change be this (in yari)?

Please note that the macros are now rendered by rari, see: https://github.com/mdn/rari/blob/03fc5598f482df7992ad81c6d25bb0a8c0f17dec/crates/rari-doc/src/templ/templs/mod.rs#L106

Nevertheless I will move this issue to yari, because we might solve this with upcoming Playground changes.
